servers/src/test/java/tachyon/worker/block/TieredBlockStoreTests.java

```diff
@@ -335,4 +336,34 @@ public void abortCommitedBlockTest() throws Exception {
     mBlockStore.commitBlock(USER_ID1, TEMP_BLOCK_ID);
     mBlockStore.abortBlock(USER_ID1, TEMP_BLOCK_ID);
   }
+
+  @Test
+  public void moveNonExistingBlockTest() throws Exception {
+    mThrown.expect(NotFoundException.class);
+    mThrown.expectMessage("Failed to get BlockMeta: blockId " + BLOCK_ID1 + " not found");
+
+    mBlockStore.moveBlock(USER_ID1, BLOCK_ID1, mTestDir1.toBlockStoreLocation());
+  }
+
+  @Test
+  public void moveTempBlockTest() throws Exception {
+    mThrown.expect(InvalidStateException.class);
+    mThrown.expectMessage("Failed to move block " + TEMP_BLOCK_ID + ": block is uncommited");
+
+    TieredBlockStoreTestUtils.createTempBlock(USER_ID1, TEMP_BLOCK_ID, BLOCK_SIZE, mTestDir1);
+    mBlockStore.moveBlock(USER_ID1, TEMP_BLOCK_ID, mTestDir2.toBlockStoreLocation());
+  }
+
+  @Ignore
```

Comment:
Sorry...But I suggest that maybe you need look through the codebase again then you will make out what "we shall never reach here" really means...